api/Node.json

```
@@ -80,7 +80,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": "9"
+              "version_added": "7"
```

Thanks for the update, @fzs111! I did a bit of digging myself and found this ancient Microsoft page which implies that appendChild existed at least as far back as IE 5 (and perhaps even partially as of IE 4) https://msdn.microsoft.com/en-us/windows/ms536389(v=vs.71). May I suggest setting this to 5?

```suggestion
              "version_added": "5"
```

---

@fzs111 do you plan on revisiting this PR?

---

I've changed this to IE 5 now.